mmcv/runner/epoch_based_runner.py

```diff
@@ -45,6 +45,7 @@ def train(self, data_loader, **kwargs):
         self.call_hook('before_train_epoch')
         time.sleep(2)  # Prevent possible deadlock during epoch transition
         for i, data_batch in enumerate(self.data_loader):
+            self.data_batch = data_batch
```

Comment:
How about deleting `self.data_batch` in line 53 to strict the scope of `runner.data_batch`

```
del self.data_batch
```